var/spack/repos/builtin/packages/perl-perlio-utf8-strict/package.py

```diff
@@ -13,3 +13,5 @@ class PerlPerlioUtf8Strict(PerlPackage):
     url = "http://search.cpan.org/CPAN/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.002.tar.gz"
 
     version("0.002", sha256="6e3163f8a2f1d276c975f21789d7a07843586d69e3e6156ffb67ef6680ceb75f")
```

Comment:
I was not able to build version 0.002 on my system but the latest version, 0.009 worked fine, so this should probably be updated.
```suggestion
    version("0.009", sha256="ba82cf144820655d6d4836d12dde65f8895a3d905aeb4aa0b421249f43284c14")
    version("0.002", sha256="6e3163f8a2f1d276c975f21789d7a07843586d69e3e6156ffb67ef6680ceb75f")
```